README.md

```diff
@@ -8,6 +8,10 @@ This is a local development harness for catalog.data.gov.
 
 Run `make build up` to build and run the docker images. 
 
+Add `127.0.0.1 ckan` to your `/etc/hosts` file
```

Comment:
Is it possible to avoid this manual step?

Reply:
No, deleted

Reply:
This change depends on your OS and requires root privileges.
The best approach here could be set _localhost:5000_ in all configs but I think we are going to have troubles using _datapusher_ and other network connections between containers.
I think it's okay to leave it as it is now.